README.md

```diff
@@ -12,87 +12,154 @@
 
 # TestResultWebApp (TRWebApp)
 
-::: {.contents local=""}
-:::
+## Table of Contents
+
+-   [Getting Started](#getting-started)
+    -   [How to install](#how-to-install)
+-   [Usage](#usage)
+-   [Contribution](#contribution)
+-   [Sourcecode Documentation](#documentation)
+-   [Feedback](#feedback)
+-   [About](#about)
+    -   [Maintainers](#maintainers)
+    -   [Contributors](#contributors)
+    -   [License](#license)
+
+## Getting Started
+
+[TRWebApp](https://github.com/test-fullautomation/testresultwebapp) is a
+generic web based open source application which allows processing
+displaying of test result data in a sophisticated way.
+
+[TRWebApp](https://github.com/test-fullautomation/testresultwebapp) uses
+a mysql data base, nodejs on server side and JavaScript on Web-Browser
+side.
+
+[TRWebApp](https://github.com/test-fullautomation/testresultwebapp) was
+initially implemented at
+![IMG_BOSCH](packagedoc/additional_docs/pictures/Boschlogo.png) [Robert
```

Comment:
Hi Ngoan,
I know, my first version was already not good :-)
The Bosch-Logo need to be aligned to the text. Please try to align it.
Thank you,
 Thomas